Style-guide/README.md

```diff
@@ -71,8 +68,8 @@ Descriptive and consistent naming makes software easier to read and understand.
 
 - striving for clarity at the call site
 - prioritizing clarity over brevity
-- using camel case (not snake case)
-- using uppercase for types (and protocols), lowercase for everything else
+- using `camelCase` (not `snake_case`)
+- using uppercase initials for types (and protocols), lowercase first-initial for everything else
 - including all needed words while omitting needless words
```

Comment:
ditto

Reply:
Swift Guidelines already have good examples, so maybe let's use it. Maybe not copy examples verbatim, but add links to specific sections. Each paragraph has its own `id` in the HTML, so it's doable. 😉 Would be super cool to have those Swift Guidelines links here!

https://swift.org/documentation/api-design-guidelines/#include-words-to-avoid-ambiguity
https://swift.org/documentation/api-design-guidelines/#follow-case-conventions